src/CustomPassword.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import './App.css';
+
+class CustomPassword extends Component {
+  render() {
+    return (
+      <div>
+        <label htmlFor="password">Password</label>
```

Comment:
What if the field is named "Credentials" or something else? this is not generic. Make sure to take the name from props also (see comment above).